src/auxiliary/t8code.jl

```diff
@@ -419,5 +647,7 @@ function trixi_t8_adapt!(mesh, indicators)
 
     mesh.forest = forest_cached
 
+    t8_forest_write_vtk(mesh.forest, "forest")
```

Comment:
```suggestion
```
Is this function supposed to always write a vtk file when it is called? Looks to me that `trixi_t8_adapt!` is called from the `AMRCallback`, where one usually would not assume some vtk output.

Reply:
good point!

Reply:
Done!

Reply:
That is indeed a mistake. Thx for the catch!